### ISSUE TITLE
Bug (inventory) maybe

### ISSUE THREAD

**Issue by Psychical** [tastybento/ASkyBlock-Bugs-N-Features#104]
I have noticed that I have an Island, taked all items to my inventory ant do a reset (/ai reset) selected new Island (while I was on my old island) and I got all items and also new items (in chest).

Photo: http://puu.sh/veBJt/ae63768dac.png

It's bug for AcidIsland or maybe my server plugins?

**Comment by Poslovitch** [comment]
I do not know how does the inventory reset on island creating works, and as @tastybento is away for about a week, i'll ask you to wait a bit.

**Comment by tastybento** [comment]
There was a bug fixed in 3.0.6.2 that stopped this exploit. It only works if you are in another world when you do a reset. Can you try and replicate with 3.0.6.2?

**Comment by Psychical** [comment]
It's already 3.0.6.2.

http://puu.sh/vfuWR/04b5bb253a.png

Yesterday, 19:07, (this issue opened on 19:17)

**Comment by tastybento** [comment]
Ok. Is this a new install? Or did you upgrade? Is the config.yml set to reset inventory? What server version are you using? Can you give me steps that replicate the issue?

**Comment by Psychical** [comment]
This is an upgrade.
Reset inventory in config.yml is true ( http://puu.sh/vfxt9/064dd88559.png )
I have 1.11.2 server

---
1. Join on sever like new person.
2. Write /ai, select 2 island.
3. Take all items from chests (in the bottom of island)
4. Get back to the top and write /ai reset, then /ai confirm.
5. You got GUI with 2 island (like in step 1), but select 1 now.
6. You got all items, also You got new items in chests.
---

This was on my server.

**Comment by tastybento** [comment]
Yes, this is a bug. It only occurs when reseting and going to the standard acid island.
